Agreements/Internal/Solent_TCSW.txt

```diff
@@ -0,0 +1,10 @@
+COPX:EGHI:*:*:NORRY:*:SOLENT:London TC SW:10000:*:^NORRY
+
+COPX:EGHI:*:TABEN:*:*:SOLENT:London TC SW:10000:*:^TABEN
+COPX:EGHH:*:TABEN:*:*:SOLENT:London TC SW:10000:*:^TABEN
+
+COPX:EGHI:*:PEPIS:CPT:*:SOLENT:London TC SW:10000:*:^PEPIS
+
+COPX:EGHI:*:GWC:*:*:SOLENT:London TC SW:10000:*:^GWC
+
+COPX:EGHH:*:*:GWC:*:SOLENT:London TC SW:10000:*:^GWC
```

Comment:
Is there a reason to make these different?

Reply:
Only becuase of my understanding at the time. Fixed now.